content/ux-strat-bootcamp/week-one/_index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Week One
+---
+
+# Welcome to Bootcamp
```

Comment:
You aren't allowing self-paced learning? Are you sure?  Also, will bootcamps always start on Mondays? Will bootcamps never have pubic holidays in them? 

One thing that I would suggest is: tell people where they should be (eg: if you aren't finished with Dashboard Analysis and Visualisation by the end of week 1, please panic. )